NextcloudTalk/CallViewController.m

```diff
@@ -1233,6 +1241,28 @@ -(void)collectionView:(UICollectionView *)collectionView willDisplayCell:(UIColl
     [participantCell.buttonsContainerView setAlpha:_isDetailedViewVisible ? 1.0 : 0.0];
 }
 
+- (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollectionViewLayout*)collectionViewLayout referenceSizeForHeaderInSection:(NSInteger)section
+{
+    if (_room.type == kNCRoomTypeGroup || _room.type == kNCRoomTypePublic) {
+        return CGSizeMake(self.collectionView.frame.size.width, collectionViewHeaderHeight);
+    } else {
+        return CGSizeMake(0, 0);
+    }
+}
+
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView viewForSupplementaryElementOfKind:(NSString *)kind atIndexPath:(NSIndexPath *)indexPath {
+    UICollectionReusableView *reusableview = nil;
+    
+    if (kind == UICollectionElementKindSectionHeader) {
+        CustomHeaderView *headerView = [collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionHeader withReuseIdentifier:@"HeaderCollectionReusableView" forIndexPath:indexPath];
+        [headerView configure];
+        headerView.backgroundColor = UIColor.blackColor;
+        [headerView setConversationNameWithName:_room.displayName];
+        reusableview = headerView;
+    }
+    return reusableview;
+}
+
```

Comment:
Not sure about placing the conversation name view as collectionView's section header.
I was thinking about a floating view on top of the collection view.
But it could work (to leave it as collectionView header) if we make it sticky and we place the toggle chat button in it.
Wat do you think @alazarevic5 ? :)

Reply:
Yes I have removed header. Instead of that I put label for title above collection view for now.